frontend/src/components/ParticipantInfo.js

```diff
@@ -72,7 +73,9 @@ const useStyles = makeStyles(theme => ({
 }))
 
 const ParticipantInfo = observer(() => {
+  // root sotre
```

Comment:
Comment doesn't add much here. its already named `rootStore` and set from `rootStoreContext` intent seems clear here.